mod/groups/actions/groups/edit.php

```diff
@@ -94,7 +94,7 @@
 }
 
 // Set group tool options
-$tool_options = elgg_get_config('group_tool_options');
+$tool_options = groups_get_group_tool_options($group);
```

Comment:
If this is a new (not yet existing) group, we don't want to pass hook handlers an unsaved group entity. Handlers generally assume truthy means a persisted entity with GUID, and falsy means new. We need to document this with the hook and for the function.